src/bunnies/random_bunny.py

```diff
@@ -8,5 +8,5 @@ def __init__(self):
         super().__init__()
 
     
-    def binarize(self, parameter):
+    def binarize(self, parameter) -> torch.Tensor:
```

Comment:
This `-> torch.Tensor` would be better placed in the abstract class no?

Reply:
Done